docs/reference/edgectl-download.md

```diff
@@ -0,0 +1,42 @@
+# Downloads
+
+You can download `edgectl`, the executable for Edge Control on a macOS, Linux, or
+Windows operating machine.
+
+On macOS and Linux, you have the option to install `edgectl` via a curl command or by downloading and executing a binary file.
+
+## macOS
+
+1. To download by curl command, run the following in your command line:
+
+```shell
+sudo curl -fL https://metriton.datawire.io/downloads/darwin/edgectl -o /usr/local/bin/edgectl && sudo chmod a+x /usr/local/bin/edgectl
+```
+
+2. Or, download the binary file [here](https://metriton.datawire.io/downloads/darwin/edgectl)
```

Comment:
I prefer not to have "here" in the prose because it doesn't read smoothly. For example, use "Or, download [the binary file]" over "Or, download the binary file [here]".